Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine as build-env
+FROM golang:1.17-alpine as build-env
```

Comment:
Bumping this because the changes to [utils/hash.go](https://github.com/worldcoin/hubble-commander/pull/625/files#diff-7f3e0a8e5f1045cea44d329f5e7695dcb70556b8d7b8e2cd26afd2dc685c04f8) use new 1.17 syntax. Our `go.mod` already declares 1.17, but this `Dockerfile` seems to be involved with our build process and it seems to have been accidentally created against an older go version.